test/integration/compliance_history_test.go

```diff
@@ -690,21 +693,23 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
 			Expect(err).ToNot(HaveOccurred())
 
 			// Delete subscription, catalog, etc at once
-			By("Deleting the namespace")
-			_, err = common.OcHub(
+			By("Deleting the namespace policy")
+			_, err = common.OcManaged(
 				"delete",
-				"ns",
-				opNs,
+				"-f",
+				nsPolicyPath,
+				"-n", policyNS,
 				"--ignore-not-found",
 			)
 			Expect(err).ToNot(HaveOccurred())
 		})
 
 		By("Creating the namespace")
-		_, err := common.OcHub(
-			"create",
-			"ns",
-			opNs,
+		_, err := common.OcManaged(
```

Comment:
Did you mean to apply this to the hub since this is a policy?

Reply:
oh mistake Thanks